docs/ug/cli.md

```diff
@@ -16,7 +16,7 @@ The command `java -jar RepoSense.jar` takes several flags.
 **Examples**:
 
 An example of a command using most parameters:<br>
-`java -jar RepoSense.jar --repo https://github.com/reposense/RepoSense.git --output ./report_folder --since 31/1/2017 --until 31/12/2018 --formats java adoc xml --view --ignore-standalone-config --last-modified-date --timezone UTC+08`
+`java -jar RepoSense.jar --repos https://github.com/reposense/RepoSense.git --output ./report_folder --since 31/1/2017 --until 31/12/2018 --formats java adoc xml --view --ignore-standalone-config --last-modified-date --timezone UTC+08`
 
 Same command as above but using most parameters in alias format:<br>
```

Comment:
This one is missing the `timezone alias - t`. Since we are trying to showcase the same command, better to update the alias command to show `t` too. 

Once, this is done you can update the PR name to - Fix errors in CLI documentation.

Reply:
Thanks for pointing it out. I have changed it accordingly.

Reply:
Need to update the commit message too accordingly.

Reply:
Sure, no problem